src/ResourceMover/examples/Get-AzResourceMoverMoveCollection.md

```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
```

Comment:
Please add related examples in the examples folder. And the examples in docs folder are generated based on the examples you provide here.

Reply:
In this case, can we remove the examples from the doc folder and check-in the docs folder? Also, how are they generated? Can we generate them using an offline process ?

Reply:
Added the examples